src/jit/vartype.h

```diff
@@ -176,7 +176,7 @@ inline bool varTypeIsI(T vt)
 template <class T>
 inline bool varTypeCanReg(T vt)
 {
-    return ((varTypeClassification[TypeGet(vt)] & (VTF_INT | VTF_I | VTF_FLT)) != 0);
+    return (TypeGet(vt) != TYP_STRUCT);
```

Comment:
It looks like this function has became a duplicate of `varTypeIsEnregisterableStruct`.

Reply:
Yes, I hadn't realized they were duplicates. I'll look into eliminating this one (I prefer the more verbose naming, though could be convinced otherwise).

Reply:
Verbose or not it seems to me that both names have some issues. `varTypeCanReg` doesn't sound quite right to me from the English language point of view (varTypeCanRegister or what?). The other one has a "Struct" suffix that seems redundant/misleading, might be nice to remove that if it's not too much churn.

Reply:
maybe just drop the struct and call it:

`varTypeIsEnregisterable`

Reply:
Yes, I've just done that. There are not so many uses that it would create a lot of churn.